src/actions/index.js

```diff
@@ -4,6 +4,7 @@ export const LOAD_ACTIVITIES = "LOAD_ACTIVITIES";
 export const LOGIN = "LOGIN";
 export const LOGOUT = "LOGOUT";
 export const REGISTER = "REGISTER";
+export const TOGGLE = "TOGGLE";
```

Comment:
I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

Reply:
removed